net-dialup/accel-ppp/files/accel-ppp-1.12.0-printf.patch

```diff
@@ -0,0 +1,16 @@
+# No need to include printf.h
+#
+# Adapted from Alpine linux patch:
```

Comment:
Was this one really adapted or taken literally?

Reply:
This is taken literally

Reply:
Then don't say "adapted" :)